tests/libnet/interface.go

```diff
@@ -0,0 +1,12 @@
+package libnet
+
+import v1 "kubevirt.io/api/core/v1"
+
+func LookupInterfaceStatusByName(interfaces []v1.VirtualMachineInstanceNetworkInterface, name string) *v1.VirtualMachineInstanceNetworkInterface {
```

Comment:
Based on the existing e2e test files, I assumed we prefer to use inside e2e test code only help functions from test package.

Reply:
I'm not seeing any added value in duplicating this helper if it is already available in an easily accessible production package. If this would have cause the import of a lot of other unrelated functionality, I would have agreed.

I guess when I suggested it, I did not noticed it already exists, sorry.

If you still wanna keep it, we can wait for another opinion about it. I won't fight over this.

Reply:
fixed